lighthouse-core/report/html/renderer/util.js

```diff
@@ -503,6 +519,9 @@ Util.UIStrings = {
   /** Label for button that only shows a few lines of the snippet when clicked */
   snippetCollapseButtonLabel: 'Collapse snippet',
 
+  /** Label shown above a list of collapsed to describe how many are in the group. The `{itemCount}` placeholder will be replaced with the number of audits, typically an integer between 1 and 30 */
```

Comment:
should we note that we don't handle plurality and they should pick the version that will just work the most often?

Reply:
didn't even realize we don't support plurality for english 😆

Reply:
Oh plurals! Yes that's why nobody had started this one yet. 🤦‍♂️

Reply:
+1 ICU Plurals for this